back/spec/factories/events.rb

```diff
@@ -2,7 +2,7 @@
 
 FactoryBot.define do
   factory :event do
-    project
```

Comment:
The default `project` factory creates a project without phases. This will make the permissions service always return an `inactive_project` disabled reason